### PR TITLE
Fat 1295 c17039

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-1295
***
https://foliotest.testrail.io/index.php?/cases/view/17039
***

### CODE REVIEW

File: cypress/e2e/data-import/field-protection-overrides.cy.js [hunk 1]
```diff
@@ -100,30 +100,30 @@ describe('ui-data-import: Check that field protection overrides work properly du
...
-    FieldMappingProfiles.deleteFieldMappingProfile(marcBibMapProfileNameForUpdate);
-    FieldMappingProfiles.deleteFieldMappingProfile(instanceMapProfileNameForUpdate);
-    FieldMappingProfiles.deleteFieldMappingProfile(marcBibMapProfileNameForUpdateAndOverride);
-    FieldMappingProfiles.deleteFieldMappingProfile(instanceMapProfileNameForUpdateAndOverride);
-
-    // delete created files
-    FileManager.deleteFile(`cypress/fixtures/${editedFileNameRev1}`);
-    FileManager.deleteFile(`cypress/fixtures/${editedFileNameRev2}`);
-  });
+  // afterEach(() => {
```
Comment (MaksimTerekhov) [inline cypress/e2e/data-import/field-protection-overrides.cy.js:103]: delete comments pls

**Review by MaksimTerekhov — COMMENTED** [review]
fix linter issues pls

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/data-import/field-protection-overrides.cy.js [hunk 2]
```diff
@@ -15,11 +15,11 @@ import TopMenu from '../../support/fragments/topMenu';
 import DataImport from '../../support/fragments/data_import/dataImport';
 import FileDetails from '../../support/fragments/data_import/logs/fileDetails';
 import Logs from '../../support/fragments/data_import/logs/logs';
-import SearchInventory from '../../support/fragments/data_import/searchInventory';
 import FileManager from '../../support/utils/fileManager';
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/field-protection-overrides.cy.js:18]: fix linter issues

File: cypress/e2e/data-import/field-protection-overrides.cy.js [hunk 3]
```diff
@@ -15,11 +15,11 @@ import TopMenu from '../../support/fragments/topMenu';
...
 import InstanceRecordView from '../../support/fragments/inventory/instanceRecordView';
 import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
 import InventoryViewSource from '../../support/fragments/inventory/inventoryViewSource';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
 
 describe('ui-data-import: Check that field protection overrides work properly during data import', () => {
   let firstFieldId = null;
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/field-protection-overrides.cy.js:25]: fix linter issues

File: cypress/support/fragments/data_import/logs/fileDetails.js [hunk 4]
```diff
@@ -101,7 +102,15 @@ export default {
...
-  openItemInInventory:(columnIndex, row = 0) => {
-    cy.do(resultsList.find(MultiColumnListCell({ row, columnIndex })).click());
+  openInstanceInInventory:(row = 0, itemStatus) => {
+    cy.do(resultsList.find(MultiColumnListCell({ row, columnIndex: 3 }))
+      .find(Link(itemStatus))
+      .click());
+  },
+
+  openHoldingsInInventory:(row = 0, itemStatus) => {
+    cy.do(resultsList.find(MultiColumnListCell({ row, columnIndex: 3 }))
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/logs/fileDetails.js:112]: Maybe MCLC you can remove to const?

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/data-import/field-protection-overrides.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/field-protection-overrides.cy.js:103]: Done

File: cypress/e2e/data-import/field-protection-overrides.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/field-protection-overrides.cy.js:18]: Done

File: cypress/e2e/data-import/field-protection-overrides.cy.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/field-protection-overrides.cy.js:25]: Done

File: cypress/support/fragments/data_import/logs/fileDetails.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/logs/fileDetails.js:112]: no, they are different

**Pull request merged**